### ISSUE TITLE
Number of glyphs

### ISSUE THREAD

**Issue by yozhic** [danpla/wdx_fontinfo#2]
## [Feature request]
The number of glyphs in the font. Is it possible to implement this? That would be very appreciated.

**Comment by danpla** [comment]
It's possible to take that information from OpenType fonts, but for other formats that don't provide that information explicitly (like some versions of PostScript fonts) this will require reading the whole file to count glyphs.

I wonder what's the use of this information? It doesn't tell how many letters are supported by the font: as I know, a glyph doesn't necessarily correspond to a letter, e.g. it can be a [diacritic](https://en.wikipedia.org/wiki/Diacritic) intended to be combined with a glyph of a letter. Neither it will tell which version of the font is newer: you have to check the version string for this, because the number of glyphs in the newer version may not only be the same, but smaller as well.

**Comment by yozhic** [comment]
_I will try to give my thoughts, but I apologize in advance, as English is not my native language._

> I wonder what's the use of this information?

In general words, it helps to quickly analyze the contents of a font without opening it in a special application. When the user sees this information, he can make the right decision faster than when he doesn't see it.  

Example 1. There is a collection of Greek fonts. Some of them support the monotonic and polytonic system, some only the monotonic system. How can I **quickly** sort them or group them for later closer examination? E.g. by the number of glyphs. Because it often happens that there is little difference in size, or that the monotonic font is even larger than the polytonic font.  

Example 2. Sometimes two files of the same font have the same version, but differ in build. If I see the number of glyphs, I will be able to understand: do I need to open the font in another program to check? E.g. if the number does not match, then of course I have to check in the other program.  

Example 3. Two files of the same font have different versions. But the number of glyphs is the same. If the developer didn't add new glyphs, why did he change the version? He probably changed the lettering. This means that I have to look more closely at this font to understand the difference.

Example 4. Some fonts are multilingual, with a huge number of glyphs - they have a large size. And others have no letters, but only decorative patterns - these are also large, but have very few glyphs.  

> other formats ... don't provide that information explicitly  

Of course, if this feature takes a lot of time to develop or a lot of resources in use, then it is not worth its cost. I'm not a coder. It's up to you to decide :-)

In short the number of glyphs is important to be seen.

**Comment by Skif-off** [comment]
@yozhic your quote is too small :)

> other formats ... don't provide that information explicitly **this will require reading the whole file**

i.e. this will require additional time. (But I agree that this can be useful information.)


I see two ways not to receive this information _every_ time:
1. Use this field only if type of font stores this information in header: OpenType is very populr format.
2. Get this informations only if user chose this field: I am not a programmer, but it seemed to me that it's not very convenient.
